tests/artist-create.test.js

```diff
@@ -0,0 +1,27 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create artist', () => {
```

Comment:
Could you add some more tests here? What happens if the artist already exists? Or if the name or genre is missing from the request?